content/services/services_development.rst

```diff
@@ -24,6 +24,51 @@ educational and government organizations looking to implement or extend open
 source software to meet their needs. Here are a few recent projects the OSL has
 worked on:
 
+Current Projects
+^^^^^^^^^^^^^^^^
+
+Here is a short (but not complete) list of current projects our development team
+is working on:
+
+`Invoicing and Metrics Project`_
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Invoicing and Metrics Project (IAM) is a web application developed at the OSL
+which collects metrics on hosted services (i.e. VMs & database usage) and tracks
+it by project. For the few projects we have hosting contracts with, this project
+allows us to easily see billable usage in a format we can put in an invoice.
+
+.. _Invoicing and Metrics Project: https://github.com/osuosl/iam
+
+`TimeSync`_
+~~~~~~~~~~~
+
+TimeSync is the OSU Open Source Lab's time tracking system. It's designed to be
+simple, have a sane API, and make sense while allowing users to track their time
+spent on various projects and activities. The project consists of a web frontend
+written in Flask (`timesync-frontend-flask`_), a backend written in NodeJS
+(`timesync-node`_) and a CLI written in Python (`climesync`_).
+
+.. _TimeSync: https://github.com/osuosl/timesync
+.. _timesync-frontend-flask: https://github.com/osuosl/timesync-frontend-flask
+.. _timesync-node: https://github.com/osuosl/timesync-node
+.. _climesync: https://github.com/osuosl/climesync
+
+`Streamwebs`_
+~~~~~~~~~~~~~
+
+TODO
```

Comment:
@Kennric I wasn't sure what to put here, can you provide a paragraph please? Also let me know if there are other projects that should be included here.